Add learning rate in XGBoost example and update readme for same (#2766)
mlflow/mlflow: examples/xgboost/README.md

## Running the code

```
python train.py --eta 0.2 --colsample-bytree 0.8 --subsample 0.9
```

> After that try experiments with different parameters like:

Reviewer: ```suggestion
You can try experimenting with different parameter values like:
```

```
python train.py --eta 0.4 --colsample-bytree 0.7 --subsample 0.8
```

> Then you can open the mlflow ui to compare your logs and track the experiments by:

Reviewer: ```suggestion
Then you can open the MLflow UI to track the experiments and compare your runs via:
```

```

> mflow ui

Reviewer: ```suggestion
mlflow ui
```
